lib/bibliothecary/parsers/conda.rb

```diff
@@ -0,0 +1,60 @@
+require 'json'
+
+module Bibliothecary
+  module Parsers
+    class Conda
+      include Bibliothecary::Analyser
+      FILE_KINDS = %w[manifest lockfile]
+
+      def self.mapping
+        {
+          match_filename("environment.yml") => {
+            parser: :parse_conda_environment,
```

Comment:
I don't think we're using `parser` at all for conda? If not, let's make it a no-op